app/Comment.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Comment extends Model
+{
+    public $timestamps = false;
```

Comment:
vidim da ih gasis. sto to? nes znati koji komentar je kad napravljen, a i generalno neki model, msm okej sad ne moras vracati samo inace je dobra praksa ostaviti